dtfs-central-api/src/drivers/db-client.js

```diff
@@ -26,8 +26,18 @@ const getConnection = async () => {
   return connection;
 };
 
+const getClient = async () => {
```

Comment:
Maybe move this below `module.exports.get = getConnection;` to the function declarations are grouped together with their `module.export`.